stable/grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 5.0.26
-appVersion: 6.7.3
+version: 5.1.0
```

Comment:
this should be
```suggestion
version: 6.0.0
```

Reply:
There are some minor breaking changes mentioned in the [release notes](https://github.com/grafana/grafana/releases). I don't think many users will be affected by any of those. I didn't have any issues with my upgrade, for example.

My [previous PR](https://github.com/helm/charts/pull/11705) to upgrade from 5.x to 6.0 was just a minor bump. I don't think it's deserving of a major chart version upgrade, but please let me know if you still think it should be and I can update it.